[2024-12-16T20:24:10.281+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data manual__2024-12-16T20:24:03+00:00 [queued]>
[2024-12-16T20:24:10.292+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.store_data manual__2024-12-16T20:24:03+00:00 [queued]>
[2024-12-16T20:24:10.293+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-16T20:24:10.308+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_data> on 2024-12-16 20:24:03+00:00
[2024-12-16T20:24:10.313+0000] {standard_task_runner.py:57} INFO - Started process 308 to run task
[2024-12-16T20:24:10.317+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'store_data', 'manual__2024-12-16T20:24:03+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmpfhpku9qg']
[2024-12-16T20:24:10.320+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask store_data
[2024-12-16T20:24:10.397+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.store_data manual__2024-12-16T20:24:03+00:00 [running]> on host 6c524df05549
[2024-12-16T20:24:10.531+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='store_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T20:24:03+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-16T20:24:03+00:00'
[2024-12-16T20:24:10.574+0000] {data_warehouse.py:50} INFO - Database tables initialized successfully
[2024-12-16T20:24:10.575+0000] {data_warehouse.py:66} INFO - Saving data to warehouse table: covid_daily_stats
[2024-12-16T20:24:10.609+0000] {data_warehouse.py:92} INFO - Successfully saved 231 records to covid_daily_stats
[2024-12-16T20:24:10.610+0000] {python.py:194} INFO - Done. Returned value was: Data stored successfully
[2024-12-16T20:24:10.654+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=store_data, execution_date=20241216T202403, start_date=20241216T202410, end_date=20241216T202410
[2024-12-16T20:24:10.705+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T20:24:10.731+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
